core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -153,8 +152,10 @@ private class ShuffleStatus(
   /**
    * Mapping from a mapId to the mapIndex, this is required to reduce the searching overhead within
    * the function updateMapOutput(mapId, bmAddress).
+   *
+   * Exposed for testing.
    */
-  private[this] val mapIdToMapIndex = new OpenHashMap[Long, Int]()
+  private[spark] val mapIdToMapIndex = new HashMap[Long, Int]()
```

Comment:
QQ: Why change to `HashMap` from `OpenHashMap` ? (it is specialized for `Long` and `Int`)

Reply:
+1 for the above question.

Reply:
`OpenHashMap` doesn't support remove operation.

Reply:
Ah yes !